copier/cli.py

```diff
@@ -158,14 +158,14 @@ def __init__(self, executable: PathLike) -> None:
         ),
     )
 
-    @cli.switch(
+    @cli.switch(  # type: ignore[misc]
         ["-d", "--data"],
         str,
         "VARIABLE=VALUE",
         list=True,
         help="Make VARIABLE available as VALUE when rendering the template",
     )
-    def data_switch(self, values: StrSeq) -> None:
+    def data_switch(self, values: Iterable[str]) -> None:
```

Comment:
Should this really be an `Iterable` and not a `Sequence`?

Reply:
It could be a sequence, but it doesn't *need* to, we only iterate on it and do not access its element by index.

Reply:
@danieleades I think it's better if you let the person asking the question mark a comment as resolved :slightly_smiling_face: